OrbitCore/OrbitModule.h

```diff
@@ -31,7 +31,7 @@ struct Module
     std::string   m_PrettyName;
     std::string   m_AddressRange;
     std::string   m_DebugSignature;
-    HMODULE       m_ModuleHandle = 0;
+    HMODULE       m_ModuleHandle = nullptr;
```

Comment:
Looking this up, it *is* a pointer, but I'm not sure you should just treat it that way. Is there a properly defined value for the default somewhere?

Reply:
This was because for Linux we simply do #define HMODULE void* in BaseTypes.h. I'll just change it back if it's confusing.